meta-refkit/classes/refkit-image.bbclass

```diff
@@ -174,6 +174,10 @@ IMAGE_INSTALL += "util-linux"
 export ALTERNATIVE_PRIORITY_SHADOW ?= "305"
 IMAGE_INSTALL += "shadow"
 
+# Install thermal management daemon by default for
+# handling platform thermal management.
+IMAGE_INSTALL += "thermald"
```

Comment:
I feel this could go ```MACHINE_EXTRA_RRECOMMENDS/RDEPENDS``` in meta-intel and we'd then install what's in that.

How to deal with the board specific configs? RMC?